### ISSUE TITLE
Do we add key for each bundled file when import?

### ISSUE THREAD

**Issue by daiduongbian44** [reactjs/React.NET#308]
We're doing with react.net and webpack, we write components for each pages then using webpack to bundle the components of each pages to get bundled page file. Suppose we have two pages (they're PAGE1, PAGE2) and we have two bundled files (they're bundle_page1.js and bundled_page2.js). We're wondering whether we do like the following source or not:
In `Reactconfig`: 
`ReactSiteConfiguration.Configuration
                .SetLoadBabel(false)
                .AddScriptWithoutTransform("bundle_page1.js", "KEY_PAGE1");`

`ReactSiteConfiguration.Configuration
                .SetLoadBabel(false)
                .AddScriptWithoutTransform("bundle_page2.js", "KEY_PAGE2");`
and in page1.cshtml:
`@Html.ReactInitJavaScript("KEY_PAGE1")`
in page2.cshtml:
`@Html.ReactInitJavaScript("KEY_PAGE2")`
If we add all components of all pages, we release the Page Loading Time is very slow but when we break all components into each pages (like the above), the Page Loading Time is decreased significantly
With current version, we think it does not support this way but we hope next version of React.Net will support for :D
Anyone has ideas?
Thanks all.
Best and regard:
Manh

**Comment by dustinsoftware** [comment]
You might be able to reference the single large bundle on the server that contains all your components, and then use a separate build step that generates a smaller bundle for each page you need.

**Comment by daiduongbian44** [comment]
@dustinsoftware you know, we're doing like your suggested way, but honestly, we want to improve speed of our site, we tested between the large bundled file for all pages and each bundled file for each page, we released the second way which it's speed is faster than the first way (using the large bundled file). So we want to break the large bundled file into set of bundled files and each pages when call a React component, we can pass the KEY that show you which bundled file the component belong to. We think that way will give our better Page Loading Time. :D thank you.
